project-management/1-1-meetings/Anne-Lee-Steele/2023-checkins-malvika.md

```diff
@@ -0,0 +1,159 @@
+## 1:1 meetings
+
+### 2023-05-18
+- Community Call: preparing agenda & getting form responses
+- Upcoming talks
```

Comment:
Most important to flag for you @malvikasharan are these upcoming schedules event schedules - to be discussed at our check-in!